app/assets/javascripts/restaurants.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Personally, I would delete these coffee script files, they won't be needed in this project
(I know they are automatically created)